### DISCUSSION TITLE
Login from Facebook not registered in auth.users

### DISCUSSION THREAD

**Discussion by auliabismar** [supabase/supabase#835] [Questions]
Is there a config I have to add in either Facebook or Supabase so the users that logged in from Facebook can be registered in auth.users table.

Thanks!

**Comment by kiwicopple** [comment]
Hey @auliabismar 👋 You will need to set up a Facebook App. (Click the "Create new credentials" on the right https://developers.facebook.com/apps/)

![image](https://user-images.githubusercontent.com/10214025/110759397-885ea800-8288-11eb-8c1f-15d36292bbf1.png)

**Reply by huy-qn** [reply]
I enabled these settings and entered the information given by Facebook but when I try to login, it failed because of the App domains are not matching. I entered `localhost` into the App domain section, but I noticed when I login the url Facebook receiving is my db url, what happened there?
